project/Dependencies.scala

```diff
@@ -16,12 +16,12 @@ object Dependencies {
   val guavaV = "28.2-jre"
   val monocleV = "2.0.0"
   val opencensusV = "0.26.0"
-  val serviceTestV = "0.18-61887ce"
+  val serviceTestV = "0.18-ae64deb-SNAP"
 
   val workbenchUtilV = "0.5-4c7acd5"
   val workbenchModelV = "0.14-2e155f0"
-  val workbenchGoogleV = "0.21-2a218f3"
-  val workbenchGoogle2V = "0.12-2187bc0"
+  val workbenchGoogleV = "0.21-64a7b29"
+  val workbenchGoogle2V = "0.13-ae64deb-SNAP"
```

Comment:
You can update to non-SNAP once the wb-libs PR is merged and travis runs against dev